src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-code">
+        Product code: 195434
+      </p>
+
+      <div class="review">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="review__text">Reviews: 5</p>
+      </div>
+
+      <div class="price">
+        <p class="price__text">Price:</p>
+        <p class="price__number">$2,199</p>
+      </div>
+
+      <button class="button">
```

Comment:
delete button and keep only a tag

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3425/files#r1293460074
Why did you resolve my comment?
I don' see changes